var/spack/repos/spack_repo/builtin/packages/petsc/package.py

```diff
@@ -580,7 +580,7 @@ def configure_options(self):
         else:
             options.append("--with-sycl=0")
 
-        if spec.satisfies("^cuda@12.8.0"):
+        if spec.satisfies("^cuda@12.8"):
```

Comment:
I'll note: this is not needed for latest petsc release (@3.23) - i.e not needed for this PR/CI [as it would get tested with latest release.]

However its good to have this for older releases -  so this could very well be in a separate PR - [or updated here]

```suggestion
        if spec.satisfies("@:3.22 ^cuda@12.8:"):

```

Reply:
My plan was to leave this change to #50284 (at a time when it seemed like this PR would go through quickly).

Reply:
I added this change to https://github.com/spack/spack/pull/50409 - the CI is clean - so I think its ready for merge - so removed it from here.